src/App.jsx

```diff
@@ -5,6 +5,7 @@ import Login from "./pages/Login";
 import Questions from "./pages/Questions";
 import Quizzes from "./pages/Quizzes";
 import Result from "./pages/Result";
+import Rules from "./pages/Rules";
 import Signup from "./pages/Signup";
```

Comment:
Since all these are coming from the pages directory only, you can make an index.js file and them from there. The code will look cleaner.

Reply:
Ok will do that after submission